api/v1alpha3/virtualmachineimage_types.go

```diff
@@ -81,6 +82,40 @@ type VirtualMachineImageProductInfo struct {
 	FullVersion string `json:"fullVersion,omitempty"`
 }
 
+// VirtualMachineImageDiskInfo describes disk information for an image taken from DiskSection
+// of an OVF Envelope.
+type VirtualMachineImageDiskInfo struct {
```

Comment:
Hey @ericvmw / @dilyar85,

I'm interested in your take on this as well -- should we be surfacing all of this detail? We definitely want the capacity and size, but I'm not sure we want _more_ than that? _Maybe_ the format? But the ID and file/parent ref information is really leaking implementation details IMO. What do y'all think?

Reply:
Removed the fields that didn't make sense